bin/rdme

```diff
@@ -20,7 +32,7 @@ require('../src')(process.argv.slice(2))
       'support@readme.io'
     )}.`;
 
-    if (err?.message) {
+    if (err.message) {
```

Comment:
Removed optional chaining from here because Node 12 wouldn't run this file at all with it (even though the `require(../src)` also loads code with it.